deploy/backplane/acs/03-acs-openshift-monitoring-rolebinding.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: backplane-acs-openshift-monitoring
+  namespace: openshift-monitoring
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:serviceaccounts:openshift-backplane-addon-acs-fleet-shard
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: backplane-acs-monitoring
```

Comment:
```suggestion
  name: backplane-acs-openshift-monitoring
```

Reply:
I'm deploying an openshift cluster as we speak to test the permissions

Reply:
Permissions have been verified on a throwaway Openshift cluster and working